backend/src/journey/service/vrrJourneysWrapper.service.ts

```diff
@@ -96,11 +99,21 @@ export class VrrJourneysWrapperService {
   }
 
   private wrapLegs(vrrLeg: VrrLeg): Leg {
+    const realtimeStatusMap: Record<RealtimeTripStatus, LegRealtimeTripStatus> = {
+      DEVIATION: LegRealtimeTripStatus.deviation,
+      TRIP_CANCELLED: LegRealtimeTripStatus.tripCancelled,
+      EXTRA_STOPS: LegRealtimeTripStatus.extraStops,
+      EXTRA_TRIP: LegRealtimeTripStatus.extraTrip,
+      MONITORED: LegRealtimeTripStatus.monitored,
+      OUTSIDE_REALTIME_WINDOW: LegRealtimeTripStatus.outsideRealtimeWindow,
+      PROGNOSIS_IMPOSSIBLE: LegRealtimeTripStatus.prognosisImpossible,
+      REALTIME_ONLY_INFORMATIVE: LegRealtimeTripStatus.realtimeOnlyInformative
+    };
```

Comment:
Needing to change this hard-coded mapping on changes to VRR's or our enum is not ideal, but I don´t have a better solution for this. Doing it this way there's no unnecessary complexity at least.